bedrock/base/templates/includes/banners/vpn-coupon-variations/vpn-coupon-promo-1.html

```diff
@@ -0,0 +1,48 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% if LANG == 'de' %}
+  {% set banner_title = "Sicher durchs Netz" %}
+  {% set banner_subtitle = "Mit 20 % Rabatt aufs Jahresabo!" %}
+  {% set banner_text = "Nutze jetzt den code <span class='c-coupon-code'>VPN20</span> beim checkout" | safe %}
```

Comment:
Since the display of this variable has `safe` it doesn't need to be here. I think it just needs to be in one place or the other, not both.